prov/efa/src/rdm/efa_rdm_ep_utils.c

```diff
@@ -580,8 +576,8 @@ ssize_t efa_rdm_ep_trigger_handshake(struct efa_rdm_ep *ep,
 
 	txe->ep = ep;
 	txe->total_len = 0;
-	txe->addr = addr;
-	txe->peer = efa_rdm_ep_get_peer(ep, txe->addr);
+	txe->addr = peer->efa_fiaddr;
```

Comment:
do you still need `addr` in txe since you already have peer?

Reply:
good idea, will fix in separate PR